tests/test_sales_tools.py

```
@@ -15,542 +15,347 @@
+        args, kwargs = mock_api_request.call_args
+        assert kwargs["params"]["filter_by"] == "open"
+        assert kwargs["params"]["date_start"] == "2023-01-01"
+        assert kwargs["params"]["date_end"] == "2023-12-31"
 
+    @pytest.mark.asyncio
+    @patch("zoho_mcp.tools.sales.zoho_api_request_async")
+    async def test_list_sales_orders_error(self, mock_api_request):
+        """Test error handling in list_sales_orders."""
+        mock_api_request.side_effect = Exception("API Error")
+        
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)